packages/cubejs-base-driver/src/BaseDriver.ts

```diff
@@ -77,6 +80,8 @@ const DB_BIG_INT_MIN = BigInt('-9223372036854775808');
 const DB_INT_MAX = 2147483647;
 const DB_INT_MIN = -2147483648;
 
+const IGNORED_DB_SCHEMAS = ['information_schema', 'mysql', 'performance_schema', 'sys', 'INFORMATION_SCHEMA'];
```

Comment:
Let's move this array to the method because it's strange to allocate an array in the module scope.